client/pages/Login/Login.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import LoginCard from '../../components/LoginCard/LoginCard';
```

Comment:
If you use package.json, the path should be "../../components/LoginCard" ... I guess